src/ewf_interface.c

```diff
@@ -933,7 +935,7 @@ ewf_result ewf_interface_drop_response(ewf_interface* interface_ptr)
     ewf_result result;
     uint8_t * response = NULL;
 
-    result = ewf_interface_receive_response(interface_ptr, &response, NULL, (uint32_t) -1);
```

Comment:
It would good to have a shorter/configurable timeout on these calls. If the modem has a crash/failure and doesn't respond then this is going to hang for a long time.